AdminApi.Sdk/Endpoints/Identities/IdentitiesEndpoint.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Backbone.AdminApi.Sdk.Endpoints.Common;
+using Backbone.AdminApi.Sdk.Endpoints.Common.Types;
+using Backbone.AdminApi.Sdk.Endpoints.Identities.Types;
+
+namespace Backbone.AdminApi.Sdk.Endpoints.Identities;
+
+public class IdentitiesEndpoint(EndpointClient client) : Endpoint(client)
+{
+    public async Task<AdminApiResponse<IndividualQuota>> CreateIndividualQuota(string address, CreateQuotaForIdentityRequest request)
+        => await _client.Post<IndividualQuota>($"Identities/{address}/Quotas", request);
+
+    public async Task<AdminApiResponse<EmptyResponse>> DeleteIndividualQuota(string address, string quotaId)
+        => await _client.Delete<EmptyResponse>($"Identities/{address}/Quotas/{quotaId}");
+
+    public async Task<AdminApiResponse<GetIdentityResponse>> GetIdentity(string address) => await _client.Get<GetIdentityResponse>($"Identities/{address}");
+
+    public async Task<AdminApiResponse<EmptyResponse>> UpdateIdentityTier(string address, UpdateIdentityTierRequest request)
+        => await _client.Put<EmptyResponse>($"Identities/{address}", request);
+
+    public async Task<AdminApiResponse<CreateIdentityResponse>> CreateIdentity(CreateIdentityRequest request)
+        => await _client.Post<CreateIdentityResponse>("Identities", request);
+
+    public async Task<AdminApiResponse<StartDeletionProcessAsSupportResponse>> StartDeletionProcessAsSupport(string address)
```

Comment:
We don't need the "AsSupport" postfix when we're already in the context of the Admin API (can you remove it in the IdentitiesController of the backend as well please?)